lib/workers/repository/onboarding/branch/rebase.ts

```diff
@@ -1,89 +0,0 @@
-import is from '@sindresorhus/is';
```

Comment:
Please, let's review and merge changes related to `rebase.ts` deletion as the separate PR

Reply:
Will that be ok. We intend to remove the rebase behaviour and replace it with merge. Don't think it will be possible to remove it and add new bheaviour in 2 PRs

Reply:
Okay, sorry for misconception then